roles/mirror_registry/README.md

```diff
@@ -0,0 +1,20 @@
+# mirror_registry
+This role can be used to deploy mirror-registry:
+https://github.com/quay/mirror-registry
+
+This is the tool used facilitate disconnected deployments of OpenShift.
+
+## Privilege escalation
+This role requires privilege escalation to facilitate the following two objectives:
+
+- It escalates privileges to create a directory under /opt that will be used to store
+  artefacts from the mirror-registry deployment; and,
+- mirror-registry leverages podman, and podman volumes. It is necessary to run podman
+  as root to create the podman volume for Quay and it's Database.
+
+## Parameters
+`cifmw_mirror_registry_basedir`: /opt/mirror-registry
+`cifmw_mirror_registry_fqdn`: quay.openstack.lab
+`cifmw_mirror_registry_release_version`: v1.3.10
+
+## Examples
```

Comment:
Not sure if this role needs an example?

Reply:
Added one, although it's basically not something I would envision used outside of CI. So maybe this needs a revisit once we know how we're going to use this role